src/api/api.js

```diff
@@ -12,4 +12,28 @@ const getFolderData = async () => {
   return result;
 };
 
-export { getUserData, getFolderData };
+const getFolderList = async () => {
+  const res = await fetch(`${API_BASE_URL}/api/users/1/folders`);
+  const result = await res.json();
+  return result;
+};
+
+const getAllCardData = async () => {
+  const res = await fetch(`${API_BASE_URL}/api/users/1/links`);
+  const result = await res.json();
+  return result;
+};
+
+const getCardDataById = async (id) => {
+  const res = await fetch(`${API_BASE_URL}/api/users/1/links?folderId=${id}`);
+  const result = await res.json();
+  return result;
+};
+
+export {
+  getUserData,
+  getFolderData,
+  getFolderList,
+  getAllCardData,
+  getCardDataById,
+};
```

Comment:
```suggestion
export const getFolderList = async () => {
  const res = await fetch(`${API_BASE_URL}/api/users/1/folders`);
  return res.json();
};

export const getAllCardData = async () => {
  const res = await fetch(`${API_BASE_URL}/api/users/1/links`);
  return res.json();
};

export const getCardDataById = async (id) => {
  const res = await fetch(`${API_BASE_URL}/api/users/1/links?folderId=${id}`);
  return res.json();
};

```
[중요도: 중간] export의 경우 꼭 최하단에 몰아서 적는 것 보다 함수 바로 옆에 적는 것을 추천드립니다.
함수가 여러개면 매번마다 밑에까지 가서 적기 귀찮으실거거든요... (지금은 괜찮아도 몇십개까지 가게 되면 적기 귀찮고 협업시에 conflict나기도 좋으니까요 ;-;)
[중요도: 중간] 
```
  const result = await res.json();
  return result;
```
에서 result를 선언하고 return하는 것은 불필요해보이네요~ 바로 res.json을 return해보는 것도 방법입니다.
[중요도: 높음] ~~result는 await res.json()으로 가져왔는데 res.json()은 비동기 처리가 아닌지라 await이 필요없습니다 ㅎㅎ~~ 이거 제 미스입니다. 비동기 맞습니다.

Reply:
74185f5e56e241c6030057c22ac22823c591bdd2 몰아서 적는 것보다 하나씩 export해주는게 더 좋군요! 수정했습니다!
47701993897b477ebeb979fb60bb1d19e8a5bf1c 이 부분도 수정했습니다!